x-pack/plugins/saved_objects_tagging/public/management/components/table.tsx

```diff
@@ -86,9 +88,11 @@ export const TagTable: FC<TagTableProps> = ({
   }
   if (capabilities.delete) {
     actions.push({
-      name: i18n.translate('xpack.savedObjectsTagging.management.table.actions.delete.title', {
-        defaultMessage: 'Delete',
-      }),
+      name: ({ name }) =>
+        i18n.translate('xpack.savedObjectsTagging.management.table.actions.delete.title', {
+          defaultMessage: 'Delete {name} tag',
+          values: { name },
+        }),
```

Comment:
Same as the `Edit` rename above, we want to avoid any repeat buttons with the same name that perform different actions. All of these buttons "Edit" or "Delete" but act on different rows so we want to make that clear.